nautobot/wireless/choices.py

```diff
@@ -0,0 +1,73 @@
+from nautobot.core.choices import ChoiceSet
+
+
+class RadioFrequencyChoices(ChoiceSet):
+    FREQUENCY_24G = "2.4 GHz"
+    FREQUENCY_5G = "5GHz"
+    FREQUENCY_6G = "6GHz"
+
+    CHOICES = (
+        (FREQUENCY_24G, "2.4 GHz"),
+        (FREQUENCY_5G, "5 GHz"),
+        (FREQUENCY_6G, "6 GHz"),
+    )
+
+
+class RadioStandardChoices(ChoiceSet):
+    A = "802.11a"
+    B = "802.11b"
+    G = "802.11g"
+    N = "802.11n"
+    AC = "802.11ac"
+    AX = "802.11ax"
+    BE = "802.11be"
+
+    CHOICES = (
+        (A, "802.11a"),
+        (B, "802.11b"),
+        (G, "802.11g"),
+        (N, "802.11n"),
+        (AC, "802.11ac"),
+        (AX, "802.11ax"),
+        (BE, "802.11be"),
+    )
```

Comment:
https://en.wikipedia.org/wiki/IEEE_802.11bn

```suggestion
class RadioStandardChoices(ChoiceSet):
    A = "802.11a"
    B = "802.11b"
    G = "802.11g"
    N = "802.11n"
    AC = "802.11ac"
    AX = "802.11ax"
    BE = "802.11be"
    BN = "802.11bn"

    CHOICES = (
        (A, "802.11a"),
        (B, "802.11b"),
        (G, "802.11g"),
        (N, "802.11n"),
        (AC, "802.11ac"),
        (AX, "802.11ax"),
        (BE, "802.11be"),
        (BN, "802.11bn"),
    )
```

Reply:
BN isn't approved yet, I purposefully left it out. But, I can add it back in.